src/pages/home/report/ReportActionsView.js

```diff
@@ -118,36 +115,14 @@ function ReportActionsView(props) {
         // eslint-disable-next-line react-hooks/exhaustive-deps
     }, []);
 
-    useEffect(() => {
-        const prevNetwork = prevNetworkRef.current;
-        // When returning from offline to online state we want to trigger a request to OpenReport which
-        // will fetch the reportActions data and mark the report as read. If the report is not fully visible
-        // then we call ReconnectToReport which only loads the reportActions data without marking the report as read.
-        const wasNetworkChangeDetected = lodashGet(prevNetwork, 'isOffline') && !lodashGet(props.network, 'isOffline');
-        if (wasNetworkChangeDetected) {
-            if (isReportFullyVisible) {
-                openReportIfNecessary();
-            } else {
-                Report.reconnect(reportID);
-            }
-        }
-        // update ref with current network state
-        prevNetworkRef.current = props.network;
-        // eslint-disable-next-line react-hooks/exhaustive-deps
-    }, [props.network, props.report, isReportFullyVisible]);
```

Comment:
Not needed anymore because `ReconnectApp` will fetch the missing updates

Reply:
Sounds right to me.